apis/v1alpha1/httproute_types.go

```diff
@@ -280,7 +280,8 @@ type HTTPPathMatch struct {
 
 	// Value of the HTTP path to match against.
 	//
-	// +kubebuilder:validation:MinLength=1
+	// +optional
+	// +kubebuilder:default="/"
```

Comment:
@nak3 since this field is now being defaulted, the `omitempty` json tag should be included.

Reply:
Thank you! Updated.